src/javascript/app/pages/trade/purchase.js

```diff
@@ -122,10 +122,9 @@ const Purchase = (() => {
 
                                 message = prepareConfirmationErrorCta(message_text, button_text, true);
                             } else {
-                                const message_text = localize('In the EU, financial binary options are only available to professional investors.');
                                 const button_text  = localize('Apply now as a professional investor');
 
-                                message = prepareConfirmationErrorCta(message_text, button_text);
+                                message = prepareConfirmationErrorCta('', button_text);
```

Comment:
I don't think we need to remove this message from purchase error. @ashkanx ?

Reply:
no, keep it. only homepage error